xbmc/epg/GUIEPGGridContainer.h

```diff
@@ -65,9 +65,8 @@ namespace EPG
     virtual std::string GetDescription() const;
     const int GetNumChannels()   { return m_channels; };
     virtual int GetSelectedItem() const;
-    const int GetSelectedChannel() const;
     CFileItemPtr GetSelectedChannelItem() const;
-    PVR::CPVRChannelPtr GetChannel(int iIndex);
+    PVR::CPVRChannelPtr GetSelectedChannel();
```

Comment:
Yeah but not in this PR. ;-) Will ake care of such things (there are many of those in this class. e.g. "const int&" parameters) in a dedicated cleanup pr.